src/main/frontend/modules/outliner/datascript.cljc

```diff
@@ -51,48 +44,32 @@
      (let [txs (remove-nil-from-transaction txs)
            txs (map (fn [m] (if (map? m)
                               (dissoc m
-                                      :block/children :block/meta :block/top? :block/bottom?
-                                      :block/title :block/body :block/level)
+                                      :block/children :block/meta :block/top? :block/bottom? :block/anchor
+                                      :block/title :block/body :block/level :block/container :level)
```

Comment:
What's the difference between :block/level and :level?

Reply:
`:level` is no longer used, I just removed it.